docs/entering-developer-mode.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Entering Developer Mode
+layout: default
+nav_order: 9
+---
+
+## Entering Developer Mode
+
+
+{: .warning }
+Entering and exiting Developer mode will **wipe all** user data!
+Backup anything if you have to.
+
+{: .warning }
+"(placeholder text) insert warning for enrolled (school, buisness) chromebooks"
```

Comment:
Can probably put something like:
"Do not continue if your device is enrolled by a organization, such as a school or employer. Unless you have consent, it is a crime."

Reply:
done